### ISSUE TITLE
Creating issues and bad votes after experience [urgent Enhancement]

### ISSUE THREAD

**Issue by SteffenVogel79** [Codewars/codewars.com#535]
I think it would be an urgend enhancement, that newbies cannot create issues or make bad votings.

In the last days it see it often, that people, who finished 1-5 katas, create issues only because they do not unterstand a kata or do not know what it means to check for null/nill.

I would recommend, that only people with 200 or more honor points can create issues or can do a bad vote.

For bad votings perhaps it should give a mandatory comment-field, in which it has to be described, why the user does a bad vote. (There are people, who think, it is funny, do give bad votes and to lower the points from the author!)

**Comment by jhoffner** [comment]
Seems reasonable to me to limit voting. 200 honor is way too high though, we will severely limit the amount of feedback and nothing will get approved. Maybe 10 - 20 honor or something, to at least force those who have little to no experience on the site to have some experience before making a judgement call. I would also say that until you reach that level of honor, you can't vote on kata at all. 

For creating issues, I don't see the big concern with allowing everyone to file an issue. The only real issue there is that the kata author isn't active on the site anymore and the issues linger, but for any active authors, they can always just close the issue.

**Comment by SteffenVogel79** [comment]
I thought of denying bad voting until 200 (or a lower frontier).
I think it is no problem, when newbies vote positive or "middle". But a bad vote is very hard. And often newbies do that, because they didn't understand a kata or have not read the description or are not very good in programming.

Would it possible only to deny bad voting until the frontier is reached. (Perhaps 50 - 100 will do instead of 200.)

I unterstand your reasons for the "issue-problem" and agree.

**Comment by jhoffner** [comment]
Yeah thats still too high of a honor limit and if you can't give your full opinion then its the equivalent of having a 5 star rating system where 70% of the users aren't allowed to give a 1 or 2 star vote. It skews the results. 

After more thought, I'm not sure how urgent this is, or if this is really even an issue. Most kata on the site have a 90% rating, very few are lower than 80%. If we put more work into propping up the scores even higher then there really is very little meaning to them since there is very little variance. If its an 8 kyu kata and beginner devs are complaining about null values, then maybe it is an issue. Maybe more of a hand holding description should be used. Or... maybe they just dont get it and the bad vote doesn't seem fare - but whatever the case they didn't enjoy the kata and they are entitled to their opinion. 

If anything I'm thinking maybe you can't vote until you have done 2 or 3 kata, so that at least you have shaken off your Codewars training wheels and have an idea of what to expect - but IDK if much more is warranted beyond that.

**Comment by SteffenVogel79** [comment]
And what about "For bad votings perhaps it should give a mandatory comment-field"?
If people have to write the reasons, some of them perhaps realize, that it is too hard.
Or otherwise the author get to know the reasons for the bad votings and could do better or improve the description...

**Comment by jhoffner** [comment]
Thats how things used to be. We had changed it because what would happen is that someone wanted to voice their opinion, and they had to comment, so you would end up with a kata that has 27 issues, and 25 of them are duplicates. So now we just strongly encourage it but dont force you.

**Comment by SteffenVogel79** [comment]
Perhaps it is really the best, if voting is possible after x done katas. But perhaps a little bit more than 2 or 3.
Maybe 10? When I think back to my beginning.... I learnt a lot of the way, like things go on codewars in the first days and the first 10 katas... ;-)

**Comment by SteffenVogel79** [comment]
For me, I think furthermoe it would be good to have a mandatory text field for bad votings.

Yesterday my Kata "https://www.codewars.com/kata/57b6f5aadb5b3d0ae3000611" was nearly 90%. So a really normal good kata. Then in 5 Minutes two guys give a bad voting.... I don't know why.... Now the kata is 81/82%. And I do not know anything. Did they hate me, because I have more points? Did they have a bad day. Is there a problem in the kata, I did not see? Or are they newbies? Or don't they care, how they vote...

**Comment by J-Sorenson** [comment]
Limiting votes until a set amount of honor or Kata completions means that  beginner Katas will never get rated or will always have a good rating (also worthless).  

If a beginner-intended Kata has poor enough instructions that many beginners down-rate it, then perhaps it is deserved.  The real issue is that some Kata may not belong in beginner section and is getting down-rated for being too difficult.

My proposal:  the user must state WHY the poor rating is deserved.  Provide a multi-select list with:
 - Poor instructions
 - Too difficult
 - Incorrect results
 - Test solution works, but always get timeouts in submission (my pet peeve: optimization Kata are not a good fit in CodeWars)
 - Other (text field)

In any case, at least the author won't be blind as to what the ratings are for.

**Comment by 10XL** [comment]
I agree with @J-Sorenson. Currently when you select 'somewhat' or 'not much' satisfaction buttons on a beta kata, there are links that encourage giving constructive feedback.

![image](https://cloud.githubusercontent.com/assets/7761011/22858664/9419b8c0-f091-11e6-9d39-8a8c40ea4f59.png)


While upon selecting the neutral/negative buttons on an approved kata, the user isn't given encouragement or guided to give feedback to authors.

![image](https://cloud.githubusercontent.com/assets/7761011/22858703/01455d54-f093-11e6-89a2-647e8f578a56.png)


Approved katas should display the same 'info box' seen on beta katas.
